### PR TITLE
fix(TESB-28966): MQTT: consume messages published before client start…

### PR DESCRIPTION
**Pull request by vdrokov**
…s up

### CODE REVIEW

File: talend-esb/pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <packaging>pom</packaging>
 
     <properties>
-        <pax-web-features.tesb.version>7.2.10.tesb1</pax-web-features.tesb.version>
```
[the remark this answers is not in the document]
Comment (vdrokov) [reply talend-esb/pom.xml]: The artifact was not found in the https://artifacts-oss.talend.com/nexus/content/repositories/TalendOpenSourceRelease as well.

[the remark this answers is not in the document]
Comment (jgreffe) [reply talend-esb/pom.xml]: The pax-web-features-7.2.10.tesb1 hasn't been pushed to artifacts-oss.
Will look into.
And I'm not sure we should set this url in pom :/

[the remark this answers is not in the document]
Comment (xldai) [reply talend-esb/pom.xml]: Okay :(,  thanks Julien. but actually it's introduced by you ~
this commit: https://github.com/Talend/tesb-rt-se/commit/9b06ee70a21cfe57ad7ffab1ca5f51caa14b5dd1#diff-90b970c5de158ecf6ada94ecb5afc92c

**Comment by vdrokov** [comment]
@xldai, @jgreffe , could you please review these changes again? I fixed the tesb1 version and locally I'm able to build without adding the custom repository now.

File: talend-esb/src/main/descriptors/win-bin.xml [hunk 2]
```diff
@@ -782,6 +782,11 @@
             <outputDirectory>/container/system/org/apache/cxf/cxf-rt-transports-http-jetty/${cxf.version}.tesb1/</outputDirectory>
             <fileMode>0644</fileMode>
         </file>
+        <file>
```
Comment (jgreffe) [inline talend-esb/src/main/descriptors/win-bin.xml:785]: Requires same dev on unix-bin.xml

Comment (vdrokov) [reply talend-esb/src/main/descriptors/win-bin.xml:785]: @jgreffe Just fixed

**Pull request merged**